website/content/docs/concepts/lease.mdx

```diff
@@ -83,3 +83,5 @@ command docs.
 This is very useful if there is an intrusion within a specific system: all
 secrets of a specific backend or a certain configured backend can be revoked
 quickly and easily.
+
+To programatically renew auth tokens, see [code example](/docs/concepts/auth#code-example).
```

Comment:
Wrong location.  Add this in the **Lease Durations and Renewal** section and not here. 

Also, I would word it like...

```suggestion
To implement token renewal logic in your application code, refer to the [code example in the Authentication doc](/docs/concepts/auth#code-example).
```